src/app/entry/entry-file-tab/state/entry-file-tab.service.ts

```diff
@@ -186,7 +187,8 @@ export class EntryFileTabService extends Base {
    */
   private setAlmostEverything(unfilteredFiles: ToolFile[]) {
     const fileTypes = this.getFileTypes(unfilteredFiles);
-    const selectedFileType: ToolFile.FileTypeEnum = fileTypes[0];
+    const previousFileType = this.entryFileTabQuery.getValue().selectedFileType;
+    const selectedFileType: ToolFile.FileTypeEnum = previousFileType ? previousFileType : fileTypes[0];
```

Comment:
Looked into this a bit (about previousFileType not applicable to this new version select). So it does indeed assign the wrong value to selectedFileType and cascades down to files and file, but some other method is able to correct this.  Regardless, it should not be doing this. It needs to check that the previousFileType is valid first.